stackslib/src/chainstate/nakamoto/coordinator/tests.rs

```diff
@@ -101,6 +102,23 @@ fn advance_to_nakamoto(
                     )
                 })
                 .collect()
+        } else if sortition_height == 7 {
+            // Vote for the aggregate key
+            test_stackers
+                .iter()
+                .enumerate()
+                .map(|(index, test_stacker)| {
+                    info!("Vote for aggregate key: {}", index);
+                    make_signers_vote_for_aggregate_public_key(
+                        &test_stacker.signer_private_key,
+                        0,
+                        index as u128,
+                        &test_signers.aggregate_public_key,
+                        0,
+                        7,
+                    )
+                })
+                .collect()
```

Comment:
This is not the correct way to get the signer indexes. We need to figure out how to get them here and sign once per signer with the correct index.